components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/cache/BaseCache.java

```diff
@@ -80,8 +80,7 @@ public BaseCache(String cacheName, int timeout, int capacity) {
             CacheManager cacheManager = Caching.getCacheManagerFactory()
                     .getCacheManager(APP_AUTH_FRAMEWORK_CACHE_MANAGER);
 
-            if (cacheTimeout > 0) {
-                if (stringCacheBuilder == null) {
+            if (cacheTimeout > 0 && stringCacheBuilder==null) {
```

Comment:
There should be spaces at either side of "==". Formatting the code will fix these.